### PR TITLE
chore: Run Markdownlint rules on fr docs - Batch 2

### PR DESCRIPTION
**Pull request by nschonni**
Continuing Markdownlint fixups

### CODE REVIEW

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 1]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
 
 On créera la piste audio avec :
 
-    ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
+```
+ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:35]: should we add `bash` code tag here:

```suggestion
```bash
ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 2]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
+```bash
+ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
+```
 
 On créera les pistes vidéos avec :
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=160:90 -b:v 250k -dash 1 video_160x90_250k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:41]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 3]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
+-an -vf scale=160:90 -b:v 250k -dash 1 video_160x90_250k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=320:180 -b:v 500k -dash 1 video_320x180_500k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:46]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 4]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
+-an -vf scale=320:180 -b:v 500k -dash 1 video_320x180_500k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=640:360 -b:v 750k -dash 1 video_640x360_750k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:51]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 5]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
+-an -vf scale=640:360 -b:v 750k -dash 1 video_640x360_750k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=640:360 -b:v 1000k -dash 1 video_640x360_1000k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:56]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 6]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
+-an -vf scale=640:360 -b:v 1000k -dash 1 video_640x360_1000k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=1280:720 -b:v 1500k -dash 1 video_1280x720_1500k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:61]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 7]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 \
-    -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=160:90 -b:v 250k -dash 1 video_160x90_250k.webm \
-    -an -vf scale=320:180 -b:v 500k -dash 1 video_320x180_500k.webm \
-    -an -vf scale=640:360 -b:v 750k -dash 1 video_640x360_750k.webm \
-    -an -vf scale=640:360 -b:v 1000k -dash 1 video_640x360_1000k.webm \
-    -an -vf scale=1280:720 -b:v 1500k -dash 1 video_1280x720_1500k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:68]:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 \
```

File: files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md [hunk 8]
```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
...
-      -f webm_dash_manifest -i video_1280x720_1500k.webm \
-      -f webm_dash_manifest -i my_audio.webm \
-      -c copy \
-      -map 0 -map 1 -map 2 -map 3 -map 4 \
-      -f webm_dash_manifest \
-      -adaptation_sets "id=0,streams=0,1,2,3 id=1,streams=4" \
-      my_video_manifest.mpd
+```
+ffmpeg \
+  -f webm_dash_manifest -i video_160x90_250k.webm \
```
Comment (yin1999) [inline files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md:81]:
```suggestion
```bash
ffmpeg \
  -f webm_dash_manifest -i video_160x90_250k.webm \
```

**Review by yin1999 — COMMENTED** [review]
and those

**Review by cw118 — APPROVED** [review]
Thank you again @nschonni 🙏 

(thanks @yin1999 for the comments)

**Pull request merged**